build.sbt

```diff
@@ -358,7 +358,9 @@ lazy val tokenizers = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .settings(
     publishableSettings,
     description := "Scalameta APIs for tokenization and their baseline implementation",
-    libraryDependencies += "com.lihaoyi" %%% "fastparse" % "1.0.0",
+    // NOTE(olafur): use shaded version of fastparse because the latest version v2.0.0
+    // has binary incompatibilites with the v1.0.0 version used by Scalameta.
+    libraryDependencies += "com.geirsson" %%% "fastparse" % "1.0.0",
```

Comment:
I can publish this under `org.scalameta` for that matter, I realised com.geirsson wasn't so suitable after the release was finished and I'm too lazy to do another release.

Reply:
If you don't mind, let's fork fastparse under the scalameta organization and publish under `org.scalameta`.

Reply:
Done, forked to scalameta/fastparse and updated the default branch to the shaded commit https://github.com/scalameta/fastparse